docs/guides/writing-a-feature-app.md

```diff
@@ -122,20 +122,31 @@ properties:
    };
    ```
 
-1. `featureAppId` — The ID that the integrator has assigned to the Feature App
-   instance. This ID is used as a consumer ID for [binding the required Feature
-   Services][feature-service-binder] to the Feature App.
+1. `featureAppId` — The ID that the integrator^1^ has assigned to the Feature
+   App instance. This ID is used as a consumer ID for [binding the required
+   Feature Services][feature-service-create] to the Feature App.
+
+1. `featureAppName` — The name that the integrator^1^ might have assigned to the
+   Feature App. This name is used as a consumer name for [binding the required
+   Feature Services][feature-service-create] to the Feature App. In contrast to
+   the `featureAppId`, the name must not be unique. It can be used by required
+   Feature Services for display purposes, logging, looking up Feature App
+   configuration meta data, etc.
 
 1. `baseUrl` — A base URL to be used for referencing the Feature App's own
-   resources. It is only set in the `env` if the integrator has defined a
+   resources. It is only set in the `env` if the integrator^1^ has defined a
    `baseUrl` on the corresponding
    [`FeatureAppLoader`][feature-app-loader-base-url] or
    [`FeatureAppContainer`][feature-app-container-base-url].
 
+1. `done` — A callback that the integrator^1^ might have defined. A short-lived
```

Comment:
we need an example for done. This is very use case specific and not as generic as the other properties.

Reply:
How about this?

> A short-lived Feature App can call this function when it has completed its task, thus giving the integrator a hint, that it can be removed. For example, if the Feature App was opened in a layer, the integrator could close the layer when `done()` is called.